flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/AccumulatorsIncludeSerializedValueQueryParameter.java

```diff
@@ -38,4 +38,10 @@ public String convertValueToString(Boolean value) {
 	public Boolean convertStringToValue(String value) {
 		return Boolean.valueOf(value);
 	}
+
+	@Override
+	public String getDescription() {
+		return "Boolean value. If true then serialized user task accumulators will be included into a response. " +
+			"False by default.";
```

Comment:
"False by default." is confusing to me, is "false": the default value of the parameter is `false`, or "false": whether the task accumulator will be included or not depends on a "default" configuration from the task accumulator?

Reply:
Done